### PR TITLE
Added doc updates for DCGM

### LINKED ISSUE #3200: As an NVIDIA user I would like better DCGM documentation/support
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I go to use the DCGMIOGroup and it is unclear how the nvidia services and GEOPM services interact

**Describe the solution you'd like**
The GEOPM service should automatically respond to the state of the NVIDIA DCGM service.  Example: when the geopm service starts the dcgm service should be started.  Similar behaviors should be taken on service start, stop, restart, etc

**Describe alternatives you've considered**
Update the DCGM documentation to explain what the user should be expected to do w.r.t. the geopm service and the nv-hostengine running on a node

### PR DESCRIPTION
**Pull request by lhlawson**
- Relates to #3200 feature request from github issues
- Fixes #3200 change request from github issues.

Updates the  documentation for better DCGM support.  
Removed initial unit file & debian control & rules for a separate PR in the future.

### CODE REVIEW

File: service/debian/control [hunk 1]
```diff
@@ -4,6 +4,7 @@ Priority: optional
 Maintainer: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>
 Build-Depends: debhelper-compat (=13),
                build-essential,
+               datacenter-gpu-manager,
```
Comment (dannosliwcd) [inline service/debian/control]: Does this also need to be in `Depends`? I think this package is combined runtime and build stuff.

Comment (lhlawson) [reply service/debian/control]: Separated the PR into documentation (the current focus) vs. implementation

File: service/docs/source/geopm_pio_dcgm.7.rst [hunk 2]
```diff
@@ -10,7 +10,16 @@ interface to provide hardware signals for NVIDIA GPUs from the NVIDIA Datacenter
 Requirements
 ^^^^^^^^^^^^
 
-To use the GEOPM DCGM signals and controls GEOPM must be compiled against the DCGM libraries and must be run on a system with hardware supported by DCGM.To compile against the oneAPI LevelZero libraries geopm must be configured using both the --enable-nvml flag and the --enable-dcgm flag.  The optional --with-dcgm flag may be used to indicate the path of the required libraries.
+To use the GEOPM DCGM signals and controls GEOPM must be compiled against the DCGM libraries and must be run on a system with hardware supported by DCGM.  
+To compile against the oneAPI LevelZero libraries geopm must be configured using both the ``--enable-nvml flag`` and the ``--enable-dcgm`` flag.
```
Comment (cmcantalupo) [inline service/docs/source/geopm_pio_dcgm.7.rst]:
```suggestion
To compile against the NVML and DGCM libraries geopm must be configured using both the ``--enable-nvml flag`` and the ``--enable-dcgm`` flag.  
```

File: service/docs/source/geopm_pio_dcgm.7.rst [hunk 3]
```diff
@@ -10,7 +10,16 @@ interface to provide hardware signals for NVIDIA GPUs from the NVIDIA Datacenter
 Requirements
 ^^^^^^^^^^^^
 
-To use the GEOPM DCGM signals and controls GEOPM must be compiled against the DCGM libraries and must be run on a system with hardware supported by DCGM.To compile against the oneAPI LevelZero libraries geopm must be configured using both the --enable-nvml flag and the --enable-dcgm flag.  The optional --with-dcgm flag may be used to indicate the path of the required libraries.
+To use the GEOPM DCGM signals and controls GEOPM must be compiled against the DCGM libraries and must be run on a system with hardware supported by DCGM.  
+To compile against the oneAPI LevelZero libraries geopm must be configured using both the ``--enable-nvml flag`` and the ``--enable-dcgm`` flag.  
+The optional ``--with-dcgm`` flag may be used to indicate the path of the required libraries.
```
Comment (cmcantalupo) [inline service/docs/source/geopm_pio_dcgm.7.rst]: I believe there is also a --with-nvml flag?

File: service/docs/source/geopm_pio_dcgm.7.rst [hunk 4]
```diff
@@ -10,7 +10,16 @@ interface to provide hardware signals for NVIDIA GPUs from the NVIDIA Datacenter
...
+
+The DCGM IOGroup requires an instance of nv-hostengine be running on the node prior to loading the IOGroup.   
+For DCGM installation and usage information see the `DCGM Getting Started Guide <https://docs.nvidia.com/datacenter/dcgm/latest/user-guide/getting-started.html>`
+The DCGM IOGroup will connect to a running nv-hostengine instance if it is available when the geopm service is started.  
+If the nv-hostengine is stopped the geopm DCGM Signals will throw an error of the form: "Error getting latest values for fields in read_batch: Host engine connection invalid/disconnected"
+Restarting the nv-hostengine and the geopm service are required to restore access to DCGM signals.
+
```
Comment (cmcantalupo) [inline service/docs/source/geopm_pio_dcgm.7.rst:48]: Maybe a note about the lines that should be added to the geopm.service unit file to support DCGM?

**Review by cmcantalupo — COMMENTED** [review]
Not as critical, but 80 character fill width would be nice as well.

**Review by cmcantalupo — APPROVED** [review]

**Pull request merged**